lib/openhab/core/sitemaps/provider.rb

```diff
@@ -49,6 +48,7 @@ def removeModelChangeListener(listener)
 
         # @!visibility private
         def unregister
+          clear
```

Comment:
Should we move this to Core::Provider so it's done for all the other providers, i.e. inform all listeners about removed elements before the script is unloaded?

Reply:
Yes!